### DISCUSSION TITLE
Default kdbx version

### DISCUSSION THREAD

**Discussion by nicoursi** [Kunzisoft/KeePassDX#1524] [Q&A]
Hi,  i noticed that if i create a new database with keepassDX it'll be kdbx3.1, but if I modify it with keepassXC it becoms kdbx4.1 and keepassDX can access and modify it correctly. 

My question is why,  if DX can work with 4.1 format,  doesn't use it as  default,  or at least allow to choose the format?

**Comment by J-Jamet** [comment]
https://keepass.info/help/kb/kdbx_4.html

It just respects the conditions of the official website. It allows to make the database file compatible with more clients. As the transition from 3.x to 4.x is based on specific data, there is no point in upgrading to 4.x if these specific data are not present in the database.
